server/package.json

```diff
@@ -25,7 +25,9 @@
     "@types/jest": "^29.5.3",
     "@types/jsonwebtoken": "^9.0.2",
     "@types/node": "^20.3.3",
+    "@types/nodemailer": "^6.4.15",
     "@types/pg": "^8.10.2",
+    "nodemailer": "^6.9.13",
```

Comment:
It should be in dependencies not in dev dependencies else it will fail in production as in production only dependencies will be considered not dev dependencies.

Reply:
Actually I tried it but the problem with this was node mailer was not recognisable until it was added to dev dependencies

Reply:
Ok, i will see this issue.